wapitiCore/parsers/swagger.py

```diff
@@ -0,0 +1,293 @@
+import json
```

Comment:
Great job. I think that file needs more explanation but looking at the tests it seems promising :+1: 

I know globally what swagger is but never put my eyes in those json/yml files.